### PR TITLE
Slide/swipe solution cards

### PR DESCRIPTION
**Pull request by gregoriusjimmy**
## Type of changes

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Enhancement
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?

- Solution cards are not swipeable

## What is the new behavior?

- Solution cards now are swipeable

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have filled the `[Unreleased]` [Changelog](https://github.com/nodefluxio/weber/blob/main/CHANGELOG.md

## Security Pentester API Checklist
- [ ] IDOR
- [ ] Malicious Script Injection
- [ ] SQL Injection

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

solutiondesktop
solutionsmobile

### CODE REVIEW

**Review by sonadztux — APPROVED** [review]
menggokil

**Comment by rizkybaihaqy** [comment]
itu tombolnya emng numpuk sama cardnya @gregoriusjimmy ?

File: frontend/app/components/templates/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -9,6 +9,12 @@ import { CardContent } from '../../modules/CardContent/CardContent'
 import { Button } from '../../elements/Button/Button'
 import Link from 'next/link'
 import { CardImage } from '../../modules/CardImage/CardImage'
+import { Navigation } from 'swiper'
+import { useMediaQuery } from '@/hooks/useMediaQuery'
+/* eslint-disable import/no-unresolved */
```
Comment (rizkybaihaqy) [inline frontend/app/components/templates/HomePage/HomePage.tsx:14]: problem eslint ini ada di hampir semua tempat tau jim

Comment (williammfu) [reply frontend/app/components/templates/HomePage/HomePage.tsx:14]: swiper ini emang ada isu pas eslint-plugin-import 😔

Comment (gregoriusjimmy) [reply frontend/app/components/templates/HomePage/HomePage.tsx:14]: ini khusus swiper doang ki dia gamau baca

**Comment by gregoriusjimmy** [comment]
> itu tombolnya emng numpuk sama cardnya @gregoriusjimmy ?

iya ki gw udah coba solve tapi ga work, karena ini kan pakai library tombolnya itu otomatis di generate di dalam container swippernya. Jadi buttonnya bakal selalu ada di paling kiri/kanan kayak gini 
image
itu gadipojok bgt karena container swippernya dibkin 80%

File: frontend/app/components/templates/HomePage/HomePage.tsx (hunk 1 above)
Comment (rizkybaihaqy) [reply frontend/app/components/templates/HomePage/HomePage.tsx:14]: owala iyak salah si packagenya kalo kata [disini](https://github.com/import-js/eslint-plugin-import/issues/2266) ma

**Review by rizkybaihaqy — APPROVED** [review]

**Review by williammfu — APPROVED** [review]

**Pull request merged**